features/steps/data/__init__.py

```diff
@@ -10,10 +10,8 @@
                       "to install the latest version.")
 from . import util
 from .plugins import HtmlOutput, Markdown
-try:
-    import tidy
-except ImportError:
-    tidy = None
+
+tidy = None
```

Comment:
I think we need to address the lack of `tidy` wholesale, rather than relying on this being `None`.